scripts/linters/pylint_extensions.py

```diff
@@ -1469,6 +1475,80 @@ def process_module(self, node):
                     line=closing_line_index_of_fileoverview + 1)
 
 
+class NewlineBelowClassDocstring(checkers.BaseChecker):
+    """Checks if there is a single newline below the class docstring."""
+
+    __implements__ = interfaces.IRawChecker
+    name = 'newline-below-class-docstring'
+    priority = -1
+    msgs = {
+        'C0026': (
+            'No single newline below class docstring.',
+            'newline-below-class-docstring',
+            'Please add a single newline below class docstring.'
+        )
+    }
+
+    def process_module(self, node):
+        """Process a module to ensure that there is a single newline below
+        class docstring.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node to access module content.
+        """
+
+        in_class = False
+        in_docstring = False
+        class_has_docstring = False
+        file_content = read_from_node(node)
+        file_length = len(file_content)
+        blank_line_counter = 0
+        prev_line = ''
+
```

Comment:
Can you please follow the pattern used in `ExcessiveEmptyLinesChecker`? I think that willmake it more simpler and consistent.

Reply:
I've tried that but think the check is not that simple, It will fail on several cases if I try that.

Reply:
Have you read the `BlankLineBelowFileOverviewChecker` checks? will that be helpful?

Reply:
Yep, I've tried that also, this check is more complicated than any of the existing ones and I can say this because last 7 or 8 checks in that file are implemented by me.